src/sentry/sentry_apps/api/bases/sentryapps.py

```diff
@@ -131,7 +128,13 @@ def handle_exception_with_details(self, request, exc, handler_context=None, scop
 
     def _handle_sentry_app_exception(self, exception: Exception):
         if isinstance(exception, SentryAppIntegratorError) or isinstance(exception, SentryAppError):
```

Comment:
are you planning on sending public context and webhook context separately in here?